packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/PropertyProvider.cs

```diff
@@ -44,6 +42,8 @@ public class PropertyProvider
 
         internal string? OriginalName { get; init; }
 
+        internal Lazy<NamedTypeSymbolProvider?>? CustomType { get; init; }
```

Comment:
Can we name this something like CustomProvider? Type makes me think it should be a CSharpType.